example/grpc_example/client/main.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I believe the source example files are redundant (not this file but the helloworld proto files)

Reply:
Without that, the example is not runnable.. ru sure?

Reply:
I thougbht so since the example file imports:
pb "google.golang.org/grpc/examples/helloworld/helloworld"